packages/core/src/common/_mixins.scss

```diff
@@ -20,10 +20,10 @@ $pt-intent-text-colors: (
 ) !default;
```

Comment:
#657 also proposes to darken intent text colors in light theme. Do we need to do that here too?